core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -95,7 +95,6 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
   override def preStart(): Unit = {
     super.preStart()
 
-    scheduleInitialMonitorPass
```

Comment:
instead of starting monitoring here, i moved it back to BootMonitors so we could easily control it from Boot.scala